### PR TITLE
build(lsp): Upgrade to flux-lsp-browser v0.5.47

### PR DESCRIPTION
**Pull request by scbrickley**
Upgrade flux-lsp-browser to [v0.5.47](https://github.com/influxdata/flux-lsp/releases/tag/v0.5.47)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -9690,6 +9690,11 @@ rsvp@^4.8.4:
   resolved "https://registry.yarnpkg.com/rsvp/-/rsvp-4.8.5.tgz#c8f155311d167f68f21e168df71ec5b083113734"
   integrity sha512-nfMOlASu9OnRJo1mbEk2cz0D56a1MBNrJ7orjRZQG10XDyuvwksKbuXNp6qa+kbn839HwjwhBzhFmdsaEAfauA==
 
+rudder-sdk-js@^1.0.16:
```
Comment (wbaker85) [inline yarn.lock:9693]: What does this do?

Comment (scbrickley) [reply yarn.lock:9693]: Huh... this PR was generated by a script, so I didn't catch that. Let me investigate and I'll get back to you.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1506)

File: yarn.lock (hunk 1 above)
Comment (scbrickley) [reply yarn.lock:9693]: @hoorayimhelping I'm not familiar with this library, and I can't think of any reason why updating the lsp would pull this in. Do you happen to have any insight into this?

Comment (scbrickley) [reply yarn.lock:9693]: Relevant github link: https://github.com/rudderlabs/rudder-sdk-js

Comment (scbrickley) [reply yarn.lock:9693]: Seems like it was a dependency that was recently introduced into the UI, but that wasn't tracked yet in the lock file.

Comment (scbrickley) [reply yarn.lock:9693]: Relevant PR: https://github.com/influxdata/ui/pull/1455

Comment (hoorayimhelping) [reply yarn.lock:9693]: Rudderstack was added with this PR: https://github.com/influxdata/ui/pull/1455. That PR should have included this yarn.lock change, but it was missed. It happens, and it's not a big deal.

This PR should fix it: https://github.com/influxdata/ui/pull/1509

If this flux-lsp change is urgent, it's not a huge deal to merge this in, it just clobbers some of the semantic meaning behind these changes.

Comment (scbrickley) [reply yarn.lock:9693]: I'll drop the change just to keep things nice and neat.

**Review by wbaker85 — APPROVED** [review]
looks good to me! I'll also be backporting this into our `OSS-2.0` branch.

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**